packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -992,6 +992,180 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         await Assertions.assertPredefinedRpcError(error, sendRawTransaction, true, relay, [signedTx, requestDetails]);
       });
 
+      it('@xts should fail "eth_sendRawTransaction" for HBAR crypto transfer to zero addresses', async function () {
+        const sendHbarTx = {
+          ...defaultLegacyTransactionData,
+          value: ONE_TINYBAR,
+          to: ethers.ZeroAddress,
+          nonce: await relay.getAccountNonce(accounts[1].address),
+          gasPrice: await relay.gasPrice(),
+        };
+
+        const signedSendHbarTx = await accounts[1].wallet.signTransaction(sendHbarTx);
+        const error = predefined.INTERNAL_ERROR('Transaction execution returns a null value');
+
+        await Assertions.assertPredefinedRpcError(error, sendRawTransaction, true, relay, [
+          signedSendHbarTx,
+          requestDetails,
+        ]);
+      });
+
+      // https://github.com/hiero-ledger/hiero-consensus-node/blob/main/hedera-node/docs/system-accounts-operations.md
+      const hederaReservedAccounts = [
+        // system accounts (≤ 0.0.750) - should return INVALID_CONTRACT_ID
+        {
+          address: '0x0000000000000000000000000000000000000002',
+          description: '0.0.2 treasury',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x0000000000000000000000000000000000000003',
+          description: '0.0.3',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x0000000000000000000000000000000000000032',
+          description: '0.0.50 system admin',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x0000000000000000000000000000000000000037',
+          description: '0.0.55 address book admin',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x0000000000000000000000000000000000000039',
+          description: '0.0.57 exchange rates admin',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x000000000000000000000000000000000000003a',
+          description: '0.0.58 freeze admin',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x000000000000000000000000000000000000003b',
+          description: '0.0.59 system delete admin',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x000000000000000000000000000000000000003c',
+          description: '0.0.60 system undelete admin',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+
+        // system contracts (precompiles) (≤ 0.0.750) - should return INVALID_CONTRACT_ID
```

Comment:
nit: use "precompiles" only for proper Ethereum precompiles to avoid confusion

```suggestion
        // system contracts (≤ 0.0.750) - should return INVALID_CONTRACT_ID
```